### PR TITLE
Login Page & endpoint + DB fix + Back Server Launch fix

### LINKED ISSUE #14: Connexion

### PR DESCRIPTION
**Pull request by Jathoosh**
Login Page & Endpoint : 
- Added login pages to both Student Front & TA Front
- Added their respective endpoints on the service auth
- As it is needed, added axios package to front
- As it is needed, added fronts' url to cors policy for the back
- Subsequently edited the DB, especially table Person, table Student & table Academic_tutor

DB fix : 
- Due to the migration to PostgreSQL from MySql, a misunderstanding in the keywords of schema and database & user, groups and role, some bugs passed under the scope of previous tests
- Fixed by specifying explicitly each needed keywords in the init.sql file
- The service auth was the only one tested during this fix, if other services do not work, add the line `schema: 'efrei_internship'` to your Sequelize option object

Back Server Launch fix : 
- In some of App.js and bin/www files of back services, there were multiple try to bind a port to the server, resulting into concurrency and so the message 'The port xxxx is already taken'
- The fix was to remove the 'listen' call in App.js, since it was already there in bin/www, fixing this issue
- Also launch.sh was updated to follow the new services and front pages

### CODE REVIEW

File: efrei-internship-bdd/init.sql [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
+SET SCHEMA 'efrei_internship';
+
+ALTER ROLE ei_user IN DATABASE db_efrei_internship SET search_path TO efrei_internship,public;
 
 create table if not exists person (
-    id_efrei int primary key,
-    password varchar(255) not null,
-    role varchar(255) not null
+    id serial primary key,
+    email varchar(255) not null,
```
Comment (RomainMarques) [inline efrei-internship-bdd/init.sql:14]: vraiment utile de stocker l'email efrei?

Comment (Jathoosh) [reply efrei-internship-bdd/init.sql:14]: Pour les intervenants ou les admin de l'efrei, pas sur qu'ils ont accès à un numéro étudiant de l'efrei pour se co (👀 déso Antoine, t'avais raison)
Egalement, pour les notifier, c'est pas mal

Comment (RomainMarques) [reply efrei-internship-bdd/init.sql:14]: faudrait modifier la table internship pour qu'elle prenne le mail du tuteur entreprise du coup non?

Comment (Jathoosh) [reply efrei-internship-bdd/init.sql:14]: Pas besoin ?
En foreign key, il est plus simple d'avoir un id. Là on a une fk sur l'id en tant que company tutor, ce qui est suffisant pour obtenir son mail par la suite

**Review by RomainMarques — APPROVED** [review]

File: efrei-internship-bdd/init.sql (hunk 1 above)
Comment (Jathoosh) [reply efrei-internship-bdd/init.sql:14]: Par contre ouais, je vais décaler "password" dans student et academic tutor ig
EDIT : Quoi que, on va considérer qu'il peut se connecter atm, et on vera par la suite

**Pull request merged**